plugin/julia-neo.vim

```diff
@@ -0,0 +1,18 @@
+function! s:RequireJuliaHost(name)
+  " Julia host arguments
+  let args = ['-e', 'using Neovim; start_host()']
+
```

Comment:
Perhaps be able to specify the host interpreter (if one has compiled master julia for instance), maybe

`let prog = get(g:, 'julia_host_prog', 'julia')`

 (in analogy with `g:python_host_prog`)